src/app/component/CookiePopup/CookiePopup.component.js

```diff
@@ -0,0 +1,55 @@
+/**
+* ScandiPWA - Progressive Web App for Magento
+*
+* Copyright © Scandiweb, Inc. All rights reserved.
+* See LICENSE for license details.
+*
+* @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+* @package scandipwa/base-theme
+* @link https://github.com/scandipwa/base-theme
+*/
+
+import { PureComponent } from 'react';
+
+import BrowserCookie from 'Util/BrowserCookie';
+import Link from 'Component/Link';
+
+import './CookiePopup.style';
+
+export const COOKIE_POPUP = 'cookie_popup';
+
+class CookiePopup extends PureComponent {
+    state = { isAccepted: BrowserCookie.getItem(COOKIE_POPUP) || false };
+
+    acceptCookies = this.acceptCookies.bind(this);
+
+    acceptCookies() {
```

Comment:
Since recently, we started to prefer such declaration:
```js
// instead of bind
acceptCookies = () => { ... }
```

It is FYI, nothing needs to be changed in this case.